tabletop/utils.py

```diff
@@ -2,9 +2,16 @@
 
 # TODO: automatically merge [secret] plugin config with secret config options in global c object
 try:
-    client = TwilioRestClient(tabletop_config['secret']['twilio_sid'], tabletop_config['secret']['twilio_token'])
+    twilio_sid = tabletop_config['secret']['twilio_sid']
+    twilio_token = tabletop_config['secret']['twilio_token']
+    client = None
+
+    if twilio_sid and twilio_token:
+        client = TwilioRestClient(twilio_sid, twilio_token)
+    else:
+        log.warning('twilio: sid and/or token is not in INI, not going to try to start twilio')
 except:
-    log.error('unable to initialize twilio rest client; you probably need to set the sid / token', exc_info=True)
+    log.error('twilio: unable to initialize twilio REST client', exc_info=True)
```

Comment:
I actually don't know when that init might raise an exception (cause I don't know anything about Twilio), I figure if it's the wrong creds, account was disabled, server issues, uber server is running without stable internet, that might cause this to throw an exception? But maybe not if it's just holding the auth data for the REST API for later usage.  I guess I had mostly assumed that if we were handling an exception there before, it would still be a good idea to handle an exception there now.